### PR TITLE
Move Allowlists to separated group

### PR DESCRIPTION
**Pull request by XaosSintez**
## Summary
This PR moves Allowlists to separated group out of Risk Score - similar to Blocklists

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by ikram-patah — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -95,6 +95,11 @@ tags:
     description: |-
       Use these operations to manage Rebilly accounts.
       An account in which you are subscribed to in order to use the Rebilly product.
+  - name: Allowlists
```
Comment (noon-dawg) [inline openapi/openapi.yaml:98]: suggestion: I think we should call this feature `Allow lists`, similar to `value lists` 
```suggestion
  - name: Allow lists
```

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -95,6 +95,11 @@ tags:
...
+    description: |-
+      Use allowlists to avoid customers from adding to blocklists.
+
+      Allowlists are lists of customer attribute values that are used to avoid [blocklist record](https://www.rebilly.com/docs/automations/blocklists/) creation when [risk score](https://www.rebilly.com/docs/automations/risk-scoring/) threshold reached.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:102]: suggestion:
```suggestion
      Use allow lists to exclude specific customer attribute data from risk score checks.
      
      Allow lists are lists of data that are excluded from risk score checks. Allow lists prevent specific data from being added to a [blocklist record](https://www.rebilly.com/docs/automations/blocklists/) when a [risk score](https://www.rebilly.com/docs/automations/risk-scoring/) threshold reached.
```

File: openapi/openapi.yaml (hunk 1 above)
Comment (XaosSintez) [reply openapi/openapi.yaml:98]: We called it allowlists similar to blocklists: https://github.com/Rebilly/api-definitions/pull/1556

File: openapi/openapi.yaml (hunk 2 above)
Comment (XaosSintez) [reply openapi/openapi.yaml]: Thank you, applied with replacing "allow list" with "allowlist"

**Review by noon-dawg — APPROVED** [review]

**Review by ikram-patah — APPROVED** [review]

**Pull request merged**